tapchannel/aux_traffic_shaper.go

```diff
@@ -277,11 +280,12 @@ func paymentBandwidthAssetUnits(htlcAssetAmount, computedLocal uint64,
 }
 
 // paymentBandwidth returns the available payment bandwidth of the channel based
-// on the list of availalbe RFQ IDs. If any of those IDs matches the channel, we
+// on the list of available RFQ IDs. If any of those IDs matches the channel, we
 // calculate the bandwidth based on its asset rate.
 func (s *AuxTrafficShaper) paymentBandwidth(htlc *rfqmsg.Htlc,
 	localBalance uint64, linkBandwidth, minHtlcAmt lnwire.MilliSatoshi,
-	fundingChan *cmsg.OpenChannel) (lnwire.MilliSatoshi, error) {
+	fundingChan *cmsg.OpenChannel,
+	peer *route.Vertex) (lnwire.MilliSatoshi, error) {
```

Comment:
The `peer` should NOT be a pointer here and below, then we can use `==` to compare (which IIRC was the whole point for adding the `route.Vertex` array type).